### PR TITLE
test solvers on CUTEstModel on Unix

### PR DESCRIPTION
**Pull request by dpo**

### CODE REVIEW

File: deps/install.jl [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-               "AmplNLReader" => "develop",
-               "Profiles" => "master")
+              "NLPModels" => "develop",
+              "OptimizationProblems" => "master",
+              "Krylov" => "develop",
+              "AmplNLReader" => "develop",
+              "Profiles" => "master")
+
+const unix_deps = Dict{AbstractString, AbstractString}(
+              "CUTEst" => "develop")
```
Comment (abelsiqueira) [inline deps/install.jl]: This requires [gfortran](https://github.com/JuliaSmoothOptimizers/CUTEst.jl/blob/develop/.travis.yml#L14).

Comment (dpo) [inline deps/install.jl]: Thanks, I just realized that too. There's so much baggage.

Comment (dpo) [inline deps/install.jl]: Should be good now.

**Comment by abelsiqueira** [comment]
Thanks

**Pull request merged**